webapp/src/DataTransferObject/Shadowing/GroupEvent.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace App\DataTransferObject\Shadowing;
+
+class GroupEvent implements EventData
+{
+    public function __construct(
+        public readonly string $id,
+        public readonly string $name,
+        public readonly ?string $icpcId,
+        public readonly ?string $type,
+        public readonly ?string $location,
+        public readonly ?bool $hidden,
+        public readonly ?int $sortorder,
+        public readonly ?string $color,
+    ) {}
```

Comment:
Probably unrelated but the fields here are not what we report in our api/doc and there we have some fields which we don't have here.

Reply:
Let's drop stuff we are not using here for now then?